src/app/components/progress-table/progress-table.component.html

```diff
@@ -170,14 +171,39 @@
         </ion-button>
       </ng-template>
     </ngx-datatable-column>
-  </ngx-datatable>
 
-  <ion-button color="primary" fill="clear" (click)="switchType()">
-    <ng-container *ngIf="type === 'team'">
-      VIEW PROGRESS BY STUDENT
-    </ng-container>
-    <ng-container *ngIf="type === 'student'">
-      VIEW PROGRESS BY PROJECT
-    </ng-container>
-  </ion-button>
+    <ngx-datatable-footer style="position: fixed; bottom: 0px;" *ngIf="true">
+      <ng-template
+        ngx-datatable-footer-template
+        let-rowCount="rowCount"
+        let-pageSize="pageSize"
+        let-selectedCount="selectedCount"
+        let-curPage="curPage"
+        let-offset="offset">
+        <ion-button class="footer-content" color="primary" fill="clear" (click)="switchType()">
+          <ng-container *ngIf="type === 'team'">
+            VIEW PROGRESS BY STUDENT
+          </ng-container>
+          <ng-container *ngIf="type === 'student'">
+            VIEW PROGRESS BY PROJECT
+          </ng-container>
+        </ion-button>
+        <div class="count footer-content">
+          {{ rowCount }} Total
```

Comment:
We already have a `count` variable in the component for the total number of records, don't need to use this `rowCount`

Reply:
yes we can use count. i will update that.